docs/view_components/welcome.rst

```diff
@@ -38,14 +38,16 @@ Requirement
 Link with other view-components
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-ref:`landing-view`:
+* ref:`landing-view`:
 
-* ref:`file menu`:
-     user can re-enable appearance of the welcome view-component through options within the file menu
+* ref:`menu_bar`:
```

Comment:
The menu bar is obviously connected to the menu bar, I guess it is a mistake?

Reply:
@Bachibouzouk 

no, no, I wanted to fix the references in welcome.rst from this branch 

so this is the text of welcome.rst

Reply:
My bad :)